packages/client/ui/react-ui/src/components/embed/crypto/CryptoEmbeddedCheckoutIFrame.tsx

```diff
@@ -27,6 +27,12 @@ export default function CryptoEmbeddedCheckoutIFrame(props: CryptoEmbeddedChecko
             console.log("[Crossmint] Received incoming transaction", serializedTransaction);
             handleIncomingTransaction(serializedTransaction);
         }
+
+        if (type === IncomingInternalEvents.CRYPTO_SWITCH_NETWORK) {
+            const { network } = payload;
+            console.log("[Crossmint] Received change of network", network);
+            (signer as ETHEmbeddedCheckoutSigner).switchNetwork!(network);
```

Comment:
```suggestion
            (signer as ETHEmbeddedCheckoutSigner).switchNetwork?(network);
```

Reply:
u might want to add a log/error if switch network isnt defined

Reply:
I meaaaan... Not really. We should only display the Dropdown experience if they send us the switchNetwork function. So if we made it to this point, it should be because the switchNetwork is not undefined. If it is, we wanna crash and know.

If we use "?" instead, if there is a bug the dev will not be receiving the notification, and we will not know. I'd rather have it crash. I can add a explicit check and throw an Error to make it explicit but I generally see those as too verbose...

Reply:
i mean a specific error is much better imo than `cannot read properties of undefined reading blah blah"`

Reply:
I have changed it, but I'm not really sure it's much more clear. To me, handling the error explicitly makes it seems like it **can happen**, and using the ``!`` makes it clear that it is expected to always exist.

I'll leave it with the error if you insist but I think it's more confusion now >_>